src/services/feed_manager.py

```
@@ -1,9 +1,8 @@
 import listparser
 from datetime import datetime
 from pathlib import Path
-from typing import Dict, List, Set, Tuple, Optional
+from typing import Dict, List, Tuple
```

Ruff (0.8.0)

`typing.Dict` is deprecated, use `dict` instead

(UP035)